### PR TITLE
Create SamuelIgwesi.json

### PR DESCRIPTION
**Pull request by SamuelIgwesi**
## Check List 

- [x ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x ] All new and existing tests passed.
- [x ] This PR does not contain plagiarized content.
- [x ] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2427"></a>

### CODE REVIEW

**Review by eddiejaoude — APPROVED** [review]
Look good 👍

File: data/SamuelIgwesi.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started Freelancing",
+      "date": "November 2021",
+      "icon": "dollar",
+      "color": "grey",
+      "description": "Started freelancing"
```
Comment (eddiejaoude) [inline data/SamuelIgwesi.json:24]: Invalid json, missing `,` here

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Looks good, but has a missing `,`

**Review by schmelto — APPROVED** [review]

File: data/SamuelIgwesi.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
+{
+  "name": "SAMUEL IGWESI",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Web Developer & Tech Enthusiast",
+  "avatar": "https://github.com/SamuelIgwesi.png",
+  "tags": ["javascript", "nodeJS"],
+   "socials": [
+    { "platform": "twitter", "url": "https://twitter.com/prime_developer" },
+    { "platform": "github", "url": "https://github.com/Samuel_Igwesi" },
```
Comment (Panquesito7) [inline data/SamuelIgwesi.json:10]: The GitHub link seems to be wrong here.
```suggestion
    { "platform": "github", "url": "https://github.com/SamuelIgwesi" },
```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

File: data/SamuelIgwesi.json (hunk 2 above)
Comment (schmelto) [reply data/SamuelIgwesi.json:10]:
```suggestion
    { "platform": "github", "url": "https://github.com/SamuelIgwesi" },
```

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**